spread/project.go

```diff
@@ -440,7 +440,7 @@ func join(scripts ...string) string {
 		}
 		buf.WriteString("(\n")
 		buf.WriteString(script)
-		buf.WriteString("\n)")
+		buf.WriteString("\n) || exit $?;\n")
```

Comment:
Having a comment and ideally a bug reference or a commit would be good, i.e. it's not obvious without that why this is needed or wanted.